snooty/test_merge_conflict.py

```diff
@@ -0,0 +1,50 @@
+import sys
```

Comment:
I'd rather this test be in `test_project.py`

Reply:
Would you be ok with two `with Project(Path("test_data/test_project"), backend) as project: [...]` blocks inside test() in `test_project.py`